tests/test_create_task.py

```diff
@@ -0,0 +1,30 @@
+import sanic
+from sanic.utils import sanic_endpoint_test
+from sanic.response import text
+from threading import Event
+import asyncio
+
+def test_create_task():
+    e = Event()
+    async def coro():
+        await asyncio.sleep(0.05)
+        e.set()
+
+    app = sanic.Sanic()
+    app.add_task(coro)
+
+    @app.route('/early')
+    def not_set(request):
+        return text(e.is_set())
+
+    @app.route('/late')
+    async def set(request):
+        await asyncio.sleep(0.1)
+        return text(e.is_set())
+
+
+    request, response = sanic_endpoint_test(app, uri='/early')
+    assert response.body == b'False'
+
```

Comment:
I do the sleep in the route, but yes that would also work or both wouldn't hurt. But, if you think about the chain of events, 0.1 in the `/late` route should always be enough. Since the first call to `sanic_endpoint_test` starts the `coro` which only sleeps for 0.05 seconds. Then this route (`/late`) sleeps for twice that time, and doesn't start until afterwards. So, I think it's sufficient length...